### ISSUE TITLE
Login by phone number

### ISSUE THREAD

**Issue by igor-imaginemage** [magento/community-features#54]
I think a good idea is provide a way to register a customer and log in by a phone number instead of email. In countries for example China people don't use emails a lot, so it's easier contact the customer by their phone.

Maybe we can provide a way to register and log in using an email or the telephone, both ways. And using the phone number we can send the transactional messages like the new order notification, shipment notification by sms messages instead of emails.

**Comment by korostii** [comment]
Wouldn't you need to rely on some kind of a 3rd-party service in order to push such SMS notifications through though? 
Do you happen to have any particular solution (for that part) in mind?

**Comment by igor-imaginemage** [comment]
Yes, we can use some third party service like Twilio (https://www.twilio.com/) to send sms messages, or with PWA we can just send push notifications.

**Comment by tomekjordan** [comment]
You can use Amasty Customer Attributes for it

**Comment by mrtuvn** [comment]
If we can login from checkout by registered phone number will be good feature i think
